pandas/core/generic.py

```diff
@@ -5635,24 +5635,68 @@ def clip(self, lower=None, upper=None, axis=None, inplace=False,
 
     def clip_upper(self, threshold, axis=None, inplace=False):
         """
-        Return copy of input with values above given value(s) truncated.
+        Return copy of the input with values above given value(s) truncated.
+
+        It truncates values above a certain threshold. Threshold can be a
+        single value or an array, in the latter case it performs the
+        truncation
+        element-wise.
 
         Parameters
         ----------
         threshold : float or array_like
+            Maximum value allowed. All values above threshold will be
+            set to this value.
         axis : int or string axis name, optional
             Align object with threshold along the given axis.
         inplace : boolean, default False
-            Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+            Whether to perform the operation in place on the data.
 
         See Also
         --------
-        clip
+        clip : Return input copy with values below/above thresholds truncated.
+        clip_lower : Return input copy input with values below given thresholds.
 
         Returns
         -------
         clipped : same type as input
+
+        Examples
+        --------
+        >>> s = pd.Series([1,2,3,4,5,6,7])
```

Comment:
PEP-8 requires space after commas. Probably with few values it's more compact,and still shows the behavior.